src/components/OwnerPanel/OwnerPanel.tsx

```diff
@@ -1,80 +1,106 @@
+          <div>${this.state.current} of ${this.state.need}</div>
+        </div>
 
-      <div className={styles.progressContainer}>
-        <div className={classnames(styles.progressBar, "progress-bar")}>
-          <div className={styles.myBar} style={{width: `${(current/need)*100}%`}}> </div>
+        <div className={styles.buttonContainer}>
+          <button className={classnames(styles.button, "button--filled")} onClick={this.showModal}>Donate</button>
+          {/* <button className={classnames(styles.button, "button--outlined")}>Get gift card</button> */}
+          <PopupModal merchant="Shunfa Bakery" option="Gift Card" className={classnames(styles.button, "button--outlined")}/>
         </div>
-        {/* pass props down here*/}
-        <div>${current} of ${need}</div>
-      </div>
 
-      <div className={styles.buttonContainer}>
-        <PopupModal merchant="Shunfa Bakery" option="Donate" className={classnames(styles.button, "button--filled")} />
-        <PopupModal merchant="Shunfa Bakery" option="Buy Gift Card" className={classnames(styles.button, "button--outlined")}/>
-      </div>
+        <ModalBox show={this.state.show} handleClose={this.hideModal}/>
 
-      <div className={styles.summaryContainer}>
-        <div>
-          <span className={styles.storeSummaryLabel}>Type: </span>
-          <span>Family-owned and operated</span>
-        </div> <br/>
-        <div>
-          <span className={styles.storeSummaryLabel}>Employees: </span>
-          <span>5</span>
-        </div> <br/>
-        <div>
-          <span className={styles.storeSummaryLabel}>Runaway: </span>
-          <span>3 months</span>
-        </div> <br/>
-        <div>
-          <span className={styles.storeSummaryLabel}>Breakeven: </span>
-          <span>$1000 / month</span>
+        <div className={styles.summaryContainer}>
+          <div>
+            <span className={styles.storeSummaryLabel}>Type: </span>
+            <span>Family-owned and operated</span>
+          </div> <br/>
+          <div>
+            <span className={styles.storeSummaryLabel}>Employees: </span>
+            <span>5</span>
+          </div> <br/>
+          <div>
+            <span className={styles.storeSummaryLabel}>Runaway: </span>
+            <span>3 months</span>
+          </div> <br/>
+          <div>
+            <span className={styles.storeSummaryLabel}>Breakeven: </span>
+            <span>$1000 / month</span>
+          </div>
         </div>
-      </div>
 
-      <div className={styles.socialContainer}>
-        {/* TEMPLATE!!! add appropriate links & social icons as needed*/}
-        <a href="#" className={classnames(styles.fa, "fa fa-twitter")} />
-        <a href="#" className={classnames(styles.fa, "fa fa-instagram")} />
-        <a href="#" className={classnames(styles.fa, "fa fa-facebook")} />
-        <a href="#" className={classnames(styles.fa, "fa fa-youtube")} />
-      </div>
+        <div className={styles.socialContainer}>
```

Comment:
nit this might be a tab vs spaces thing, but we should decide on one or the other